### PR TITLE
support for xkey in the authcallout configuration

### PR DESCRIPTION
**Pull request by aricart**
- [FEAT] added `--curve`, `--rm-curve` key options to `authcallout` command
- [FEAT] updated describer to display curve key
- [FEAT] updated key list to show curve keys associated with an account

### CODE REVIEW

File: cmd/describeaccount_test.go [hunk 1]
```diff
@@ -226,13 +226,16 @@ func TestDescribeAccount_Callout(t *testing.T) {
 
 	_, uPK, _ := CreateUserKey(t)
 	_, aPK, _ := CreateAccountKey(t)
+	_, xPK, _ := CreateNkey(t, nkeys.PrefixByteCurve)
```
Comment (derekcollison) [inline cmd/describeaccount_test.go:229]: CreateCurveKey()

**Review by derekcollison — COMMENTED** [review]
Generally looks good. I wonder if we should move curve -> x25519? Maybe curve too generic?

File: cmd/describeaccount_test.go (hunk 1 above)
Comment (aricart) [reply cmd/describeaccount_test.go]: Now that the nkey api correctly dispatches to the right API this is safe.

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**